CMakeLists.txt

```diff
@@ -455,6 +455,11 @@ add_subdirectory(${FLATBUFFERS_SRC_DIR}
         ${CMAKE_CURRENT_BINARY_DIR}/flatbuffers-build
         EXCLUDE_FROM_ALL)
 
+set(FMT_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/third_party/fmt-9.1.0/include")
```

Comment:
Btw fmt promises to have super fast algorithm for floating point numbers rendering https://github.com/fmtlib/dtoa-benchmark#results

We can try to re-use it in our JSON renderer here https://github.com/Project-OSRM/osrm-backend/blob/ef8f3d7508dbdcaeac990d1b891d2e0775c594e6/include/util/json_renderer.hpp#L41